lib/shoulda/matchers/active_record/association_matcher.rb

```diff
@@ -990,6 +990,13 @@ class AssociationMatcher
 
         attr_reader :name, :options
 
+        MACROS = {
```

Comment:
Would you mind putting this at the top of the class? I believe that we have been pretty good about putting them first.